fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/typing/Type.java

```diff
@@ -1159,6 +1160,11 @@ public int getFieldIndex() {
                 return getFieldIndexOptional().orElseThrow(() -> new RecordCoreException("field index should have been set"));
             }
 
+            @Nonnull
+            public KeyExpression toKeyExpression() {
```

Comment:
This seems odd to have that here as a `KeyExpression` is a thing that computes something and a `Type` is something that tells you what a. thing looks like. By the same rational you could have a `toFieldValue()` here which is equally odd.

Reply:
You're right, I removed this method from both `Type.Record.Field` and `FieldValue`.